general/clients/index.md

```diff
@@ -208,13 +208,14 @@ The official Jellyfin WebOS app.
 
 ### Mopidy-Jellyfin
 
-A third party plugin for Mopidy that uses Jellyfin as a backend.
+An official plugin for Mopidy that uses Jellyfin as a backend.
 
-**Status:** ⭐ Active, 3rd-Party
+**Status:** ⭐ Active
 
 **Links:**
 
 * [GitHub](https://github.com/mcarlton00/mopidy-jellyfin)
+* [Installing](xref:clients-installing-mopidy)
```

Comment:
Could you add the link above for Kodi as well? It's a good idea.

Reply:
While technically not within the scope of this PR, it's not worth it's own.  done.